hack/junit-util.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# junit-util builds the jUnit report parsing bianries and sets up the output directories for 
```

Comment:
provide a method in here that takes the `go test` verbose output and writes out a junit file.  That will hide almost all the ugly.